dime-research-standards/pillar-3-research-reproducibility/checklists/tex/reproducibility-package-guidance-note.tex

```diff
@@ -0,0 +1,109 @@
+	\end{itemize}
+
+	DIME Analytics will only change the topmost global directory specified in the master script and will run it to reproduce the results – this is what is called a one-button run. The master script code should run without errors after adding the correct folder path of the computer where it will be running. For the paper to be considered fully reproducible, DIME Analytics should reproduce the every exhibit in the paper showing analysis results through this push-button exercise.
+
+	\bigskip
+
+	After receiving a reproducibility package, DIME Analytics will attempt a first one-button run. If the master script breaks, DIME Analytics will identify the point where it is breaking, document it, and attempt to make changes to continue with the review. If the corrections cannot be identified after a 15-minute check, the project team will be notified that further edits are required to conduct the reproducibility check.
+
+	\bigskip
+
+	If the code runs from beginning to end, DIME Analytics will send the team an email confirming that we were able to run the code and providing an estimated time to deliver the results of the review. Reproducibility checks take ten working days (two weeks) in most cases, but might take longer for computationally intensive packages, packages with more than 1 GB of total file size, or when a large volume of submissions is received.
+
+	\bigskip
+	
+	\begin{center}
+		\includegraphics[width=0.9\linewidth]{../img/rep-checks-timeline.png}
+	\end{center}
+
+	\subsection{2. \, Reproducibility check}
+
+	After the one-button run works, DIME Analytics will check the following code requirements. The submission should pass all of these checks to be in compliance with DIME Reproducibility Standards.
+
+	\bigskip
+
+	\begin{itemize}
+		\setlength\itemsep{-0.1em}
+		\item \textbf{Code completeness:} Every paper exhibit containing analysis results should be produced programatically by the code and be a result of statistical analysis. Manual entry of values used in a table or plot, or point-and-click-generated exhibits are not in compliance with DIME Reproducibility Standards.
+		\item \textbf{Output stability:} DIME Analytics will run the code between three to five times. Every output should be  identical across each run.
+		\item \textbf{Computational reproducibility of tables:} Every statistic of every table needs to be consistent between the outputs produced by the code and the exhibits of the paper. Occasionally, stable marginal differences in point estimates and standard errors might not break adherence with DIME Reproducibility Standards and will be evaluated in a case-by-case basis to determine if they can occur for rounding differences due to hardware or OS specifications.
```

Comment:
should we set a specific standard like differences of less than 0.001 will be noted but not rejected?

Reply:
I'd prefer to say that we'll evaluate them case-by-case. Differences in that magnitude can still be due to code instabilities.  Nonetheless, I'm changing this text from "stable marginal differences" to "stable differences of less than 0.001"